src/main/resources/migration/common/V11__add_prison_id_column_to_assessment_table.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE assessment ADD COLUMN prison_id VARCHAR(3) NOT NULL;
```

Comment:
I didn't think this would work because you are adding a new not null column without a default value but it looks it does. I guess it's because we don't have any entries.